Dockerfile

```diff
@@ -135,16 +135,12 @@ RUN curl -fsSL https://github.com/iqtree/iqtree2/releases/download/v2.1.2/iqtree
 COPY builder-scripts/ /builder-scripts/
 
 # Download Nextalign v2
-# Set default Nextalign version to 2
 RUN curl -fsSL https://github.com/nextstrain/nextclade/releases/download/2.14.0/nextalign-$(/builder-scripts/target-triple) \
-  -o /final/bin/nextalign2 \
-  && ln -sv nextalign2 /final/bin/nextalign
```

Comment:
Assuming pathogen-repo-ci runs nextalign on the CI example data, I would expect it to fail on the following which still use Nextalign CLI:

- [ncov](https://github.com/nextstrain/ncov/blob/f79bb86c0ba20f685a877e8fa7fbe803ddcf2d9f/workflow/snakemake_rules/main_workflow.smk#L95)
- [seasonal-flu](https://github.com/nextstrain/seasonal-flu/blob/d149b2de52620b8f6f8b675e0215e3cbe01e79e7/workflow/snakemake_rules/core.smk#L69)

Reply:
Good point! The lines should stay then!

Reply:
Shouldn't the workflows be updated to either use `nextalign2` explicitly or replace with v3 Nextclade CLI?

Reply:
Oh nevermind, that's not necessary - runtimes will keep `nextalign` around for backwards compatibility. `nextalign` is unambiguously v2 as latest, unlike `nextclade`.

Reply:
Added the lines back in [force-push](https://github.com/nextstrain/docker-base/compare/61a8af29ffa95a9ae9f4aff7c42cee9217ad0c8f..ee302be584707bc1d391931fb8bb2a68ba87d620).